front/src/applications/stdcm/components/StdcmResults/StdcmSimulationNavigator.tsx

```diff
@@ -27,7 +28,7 @@ const StdcmSimulationNavigator = ({
   isCalculationFailed,
   onSelectSimulation,
 }: StdcmSimulationNavigatorProps) => {
-  const { t } = useTranslation('stdcm', { keyPrefix: 'simulation.results' });
+  const { t } = useTranslation();
```

Comment:
I initially kept it this way because I was translating duration units directly in the component, but I later moved the translation inside the formatting function.

Now, I’m leaving it as is because I added 'Km' to the translation dictionary.